packages/components/src/animations/fade-slide.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Used by
+ * 1. PostHeader (mobile)
+ * 2. PostMegadropdown
+ */
+
+import { CurveEasing, PresetEasing, AnimationOptions } from './types';
+
+const defaultOptions: AnimationOptions = {
+  translate: -10,
+  duration: 300,
+  easing: 'linear',
```

Comment:
Can it default to the header animation duration and easing? If I am not mistaken this is never used with 300ms linear.
Then you can remove this file: `packages/components/src/animations/presets.ts`

```suggestion
  duration: 350,
  easing: {
    x1: 0.8,
    y1: 0.2,
    x2: 0.8,
    y2: 0.7,
  },
```